packages/firestore/src/local/simple_db.ts

```diff
@@ -35,10 +36,12 @@ const LOG_TAG = 'SimpleDb';
 export class SimpleDb {
   /** Opens the specified database, creating or upgrading it if necessary. */
   static openOrCreate(
+    databaseId: DatabaseId,
     name: string,
     version: number,
     runUpgrade: (
       db: IDBDatabase,
+      databaseId: DatabaseId,
```

Comment:
SimpleDb is something we have previously discussed making into a separate package from Firestore so we should try to keep Firestore-specific concepts like "databaseId" out of the SimpleDb API.